.github/workflows/deploy-happy-stack.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     steps:
       - uses: actions/setup-node@v2
```

Comment:
and do we need to bump this version to 4 or is it not blocking?

Reply:
non-blocking